engine/swarm/ingress.md

```diff
@@ -101,7 +101,7 @@ you publish both TCP and UDP ports, If you omit the protocol specifier,
 the port is published as a TCP port. If you use the longer syntax (recommended),
 set the `protocol` key to either `tcp` or `udp`.
 
-#### TCP only
+#### TCP and UDP
```

Comment:
@mahs4d The existing heading looks correct as the following example demonstrates the default behavior, which is TCP only.

Reply:
@usha-mandya you are right, i didn't know the default behavior :) i'll remove the PR